src/gui/editors/SongEditor.cpp

```diff
@@ -98,6 +98,10 @@ SongEditor::SongEditor( Song * song ) :
 	connect( m_timeLine, SIGNAL(selectionFinished()),
 			 this, SLOT(stopRubberBand()));
 
+	// when tracks realign, adjust height of position line
```

Comment:
Prefer the new connect syntax:

    connect(this, &TrackContainerView::trackRealigned, this, &SongEditor::updatePositionLine)